alfred/Makefile

```diff
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=alfred
-PKG_VERSION:=2019.5
+PKG_VERSION:=2020.0
 PKG_RELEASE:=0
```

Comment:
Strictly, PKG_RELEASE starts at "1", not at "0". Since we are changing PKG_VERSION here, which would normally trigger PKG_RELEASE _reset_ to "1", I'd be happy if you could update PKG_RELEASE to 1 as well (for all three packages).

If you do so, please also add one (trivial) line of explanation to the commit messages ...

Reply:
The PKG_RELEASE number change is in another commit. The commit message contains the changes for this release